packages/cdktf-cli/lib/get/base.ts

```diff
@@ -101,5 +103,18 @@ export abstract class GetBase {
     }
   }
 
+  private static sanitizeSource(source: string, language: Language, isModule: boolean): string {
+    switch (language) {
+      case Language.JAVA:
+        return !isModule && source === "null" ? "nullprovider" : source.replace(/\//gi, '.').replace(/-/gi, '_');
+      case Language.CSHARP:
+        return !isModule && source === "null" ? "Providers.Null" : source.replace(/\//gi, '.').replace(/-/gi, '_');
+      case Language.PYTHON:
+        return source.replace(/\//gi, '.').replace(/-/gi, '_');
```

Comment:
While we're at it, perhaps extract this in a function?

Reply:
Not sure it's the best name, but I pulled it out.